assets/scripts/elfish.js

```diff
@@ -85,6 +85,9 @@ function reloadDataIntoDom() {
 
     // Updates the method selector, reflecting the stored value.
     setMethodDropdown(window.elfish.method);
```

Comment:
to be renamed updateMethod[Dropdown]. It's not setting anything as far as the controller is concerned.